lib/akasha/event_router_base.rb

```diff
@@ -3,7 +3,7 @@ module Akasha
   class EventRouterBase
     def initialize(routes = {})
       @routes = Hash.new { |hash, key| hash[key] = [] }
-      @routes.merge!(routes)
+      @routes.merge!(routes.map { |command, listeners| [command, Array(listeners)] }.to_h)
```

Comment:
Would be good to add a comment, e.g. "Support both array of listeners and a single listener" or sth.

Reply:
I was thinking about it, but we already had this feature, but broken. I just fixed it.

Reply:
I mean a comment above this line so it's clear what it does. ❓

Reply:
I was thinking about changelog, sorry!